src/pages/en/guides/integrations-guide.md

```diff
@@ -0,0 +1,111 @@
+
+- Unlock React, Vue, Svelte, Solid, and other popular UI frameworks.
+- Integrate tools like Tailwind, Turbolinks, and Partytown with a few lines of code.
+- Add new features to your project, like automatic sitemap generation.
+- Write custom code that hooks into the build process, dev server, and more.
+
+> Integration support was recently released in v0.25.0, and the API is still being finalized. Only first-party Astro integrations (those published to `@astrojs/` on npm) will be officially supported during this period. To use a 3rd-party plugin, you need to run Astro with the `--experimental-integrations` CLI flag.
+## Tutorial: Adding React
+
+In this example, we will add the `@astrojs/react` integration to add React support to your Astro project. 
+
+First, you will need to install the integration and any other dependencies that you'll need. For React, that means starting by installing the `@astrojs/react` integration
+
+```bash
+npm install --save-dev @astrojs/react
+```
+
+Once your packages have been installed, add two new lines to your `astro.config.mjs` project configuration file. 
+
+```diff
+  // astro.config.mjs
+  import { defineConfig } from 'astro/config';
++ import react from '@astrojs/react';
+
+  export default defineConfig({
++   integrations: [react()],
+  });
+``` 
+
+The first line is the import statement that imports the integration into your configuration file. The second line adds that integration into your project configuration so that Astro knows to use it.
+
+That's it! Restart Astro, and the new integration should take effect immediately. If you see an error on startup, make sure that you installed the required packages with npm., and that you called your integration as a function (`good: [react()]`) and didn't pass it directly in your config (`bad: [react]`).
+
+## Peer Dependencies Warning
+
+When installing an integration from npm, keep an eye out for any "missing peer dependencies" messages during the install step. You may also need those packages in your project and not all package managers will install them for you automatically. 
+
+React, for example, is a peer dependency of the `@astrojs/react` integration. npm (v7+) will install the required React packages for you automatically when you install `@astrojs/react`. In other package managers (including older versions of npm) you will need to install these yourself, seperately:
+
+```bash
+# Install any missing peer dependencies for an integration
+npm install --save-dev react react-dom
+```
+
+If you miss this step, don't worry, Astro will warn you during startup if any missing peer dependencies are required but not found in your project.
+
+Managing your own peer dependencies may require more work, but it also lets you control what versions of packages you use for things like React, Tailwind, and more.
+
```

Comment:
I would personally reorder the flow of this section, empahsizing the benefit of managing peer dependencies first, then you'll get a warning, the what to do, then an example at the end. Just a personal preference, but something like:

Managing your own peer dependencies may require more work, but it also lets you control what versions of packages you use for things like React, Tailwind, and more.

Astro will warn you during start up if any missing peer dependencies are required, but not found in your project. If you see any "missing peer dependencies" messages during the install step, you may also need to install those packages in your project.  (Not all package managers will install them for you automatically.)

React, for example, is a peer dependency of the `@astrojs/react` integration. npm (v7+) will install the required React packages for you automatically when you install `@astrojs/react`. In other package managers (including older versions of npm) you will need to install these yourself, seperately:

```bash
# Install any missing peer dependencies for an integration
npm install --save-dev react react-dom
```

Reply:
Ah, now that the peer dependency this has been determined to be more of a "react to this issue" than "hey, just so you know"... I get why maybe addressing the error message right away makes more sense.  :) I'm OK with the original flow, since it's more reactive to the error message!